testutil/integration/evmos/network/config.go

```diff
@@ -21,20 +21,29 @@ import (
 	evmtypes "github.com/evmos/evmos/v20/x/evm/types"
 )
 
+// defaultChain represents the default EIP155 chain ID used in the suite setup.
+const defaultChain string = utils.MainnetChainID
+
 // Config defines the configuration for a chain.
 // It allows for customization of the network to adjust to
 // testing needs.
 type Config struct {
-	chainID            string
-	eip155ChainID      *big.Int
-	amountOfValidators int
-	preFundedAccounts  []sdktypes.AccAddress
-	balances           []banktypes.Balance
-	chainCoins         ChainCoins
+	chainID       string
+	eip155ChainID *big.Int
+
 	customGenesisState CustomGenesisState
-	otherCoinDenom     []string
-	operatorsAddrs     []sdktypes.AccAddress
-	customBaseAppOpts  []func(*baseapp.BaseApp)
+
+	customBaseAppOpts []func(*baseapp.BaseApp)
+
+	amountOfValidators  int
+	operatorsAddrs      []sdktypes.AccAddress
+	initialBondedAmount math.Int
+
+	chainCoins        ChainCoins
+	initialAmounts    InitialAmounts
+	otherCoinDenom    []string
```

Comment:
what is this `otherCoinDenom`? Not fully clear to me

Reply:
Added comment to clarify this, but it is a piece of code coming from the original suite. Basically are other coins that can be specified for genesis account configurations